bazelisk.go

```diff
@@ -566,6 +579,41 @@ func downloadBazel(fork string, version string, isCommit bool, directory string)
 	return destinationPath, nil
 }
 
+func copyFile(src, dst string, perm os.FileMode) (err error) {
+	data, err := ioutil.ReadFile(src)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(dst, data, perm)
+}
+
+func linkLocalBazel(directory string, bazelPath string) (string, error) {
+	directoryName := dirForURL(bazelPath)
+	destinationDir := filepath.Join(directory, directoryName, "bin")
+	err := os.MkdirAll(destinationDir, 0755)
+	if err != nil {
+		return "", fmt.Errorf("could not create directory %s: %v", destinationDir, err)
+	}
+	destinationPath := filepath.Join(destinationDir, "bazel"+determineExecutableFilenameSuffix())
+	if _, err := os.Stat(destinationPath); err != nil {
+		err = os.Symlink(bazelPath, destinationPath)
+		if runtime.GOOS == "windows" {
+			// If can't create Symlink on Windows, fallback to copy
+			if err != nil {
+				err = copyFile(bazelPath, destinationPath, 0755)
+				if err != nil {
+					return "", fmt.Errorf("cound not copy file from %s to %s: %v", bazelPath, destinationPath, err)
+				}
+			}
+		} else {
+			if err != nil {
```

Comment:
We could also use the fallback on other platforms.

Reply:
+1. First I wanted to say that symlinks always work on non-Windows platforms, but then I remembered the time where someone tried to run Bazel on Linux with $HOME being on a `vfat` volume...

I think it's fine to just always fallback to copy.